package/MDAnalysis/analysis/encore/dimensionality_reduction/src/spe.c

```diff
@@ -22,10 +22,12 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-//#include <math.h>
+#include <math.h>
 #include <time.h>
 #include <sys/types.h>
 #include <time.h>
+#include <sys/types.h>
+#include <unistd.h>
```

Comment:
The `sys/types.h` header is repeated twice. See line 27. 

Why are you including `unistd.h`? I don't see you using types defined in that header.

Why include `math.h` again?